### ISSUE TITLE
gesture object and gesture.stopDetect() not available?

### ISSUE THREAD

**Issue by iDVB** [JedWatson/react-hammerjs#59]
I may be going down the wrong path but so far as I can tell, the gesture experience I'm going for in more like [this codepen](https://codepen.io/2ne/pen/Lueit/?editors=0010), then like[ this one](http://codepen.io/iDVB/pen/yVXmgp?editors=0010).

The first one is very responsive to swipes and pans. you can flick the slides back and forth very quickly. This is also very much like what the swiper lib can do. ([seen on homepage)](http://idangero.us/swiper/#.WDh0_aIrJTY)

When I dive into the the code of that ideal example, I can see that they are using **gesture.stopDetect()** in swipes to prevent pans.

I can't seem to find the gesture object anywhere in any of the events that react-hammerjs gives back.

In short, I really just want to reproduce that first example with my carousel in react and it seems like I may have to use native hammerjs to do so?
